pydocusign/client.py

```diff
@@ -47,6 +48,8 @@ def __init__(self,
             self.account_url = '{root}/accounts/{account}'.format(
                 root=self.root_url,
                 account=self.account_id)
+        #: Requests timeout
```

Comment:
Add a period at the end of the docstring.

See [PEP-257] (http://legacy.python.org/dev/peps/pep-0257/#one-line-docstrings):

> The docstring is a phrase ending in a period.